README.md

```diff
@@ -15,6 +15,49 @@ def deps do
 end
 ```
 
+## Usage
+
+If the data is valid, the function returns {:ok, data}. The data returned will be coerced according to the schema defined.
+
+If the data passed does not follow the rules defined in the schema, the function returns {:error, error_message}. It will also return error, in case we have a field in data, whose rule is not defined in the schema.
```

Comment:
Can we also wrap the {:error, error_message} in backticks: `{:error, error_message}`.

Reply:
let's make this "It will also return an error when receiving a field that has not been specified in the provided schema"